include/realizations/config/config.hpp

```diff
@@ -0,0 +1,63 @@
+#ifndef NGEN_REALIZATION_CONFIG_H
+#define NGEN_REALIZATION_CONFIG_H
+
+#include <boost/property_tree/ptree.hpp>
+
+#include "formulation.hpp"
+#include "forcing.hpp"
+
+#include<iostream>
```

Comment:
```suggestion

```

Reply:
fixed in 3949a34da709b2fda198b15a7732928f7eac996f